Security/Provider/RefreshTokenProvider.php

```diff
@@ -129,6 +129,6 @@ public function supportsClass($class)
             return true;
         }
 
-        return User::class === $class;
+        return class_exists(User::class) && User::class === $class;
```

Comment:
SF6 drops the User class, so technically the provider shouldn't support the class if it's not available.